wee_slack.py

```diff
@@ -3104,6 +3144,7 @@ def tag(tagset, user=None):
     tagsets = {
         # messages in the team/server buffer, e.g. "new channel created"
         "team": "no_highlight,log3",
+        "team_highlight": "irc_privmsg,notify_message,log1",
```

Comment:
This is not a necessarily a highlight, just a normal notify that allows highlights. You won't get highlighted by any of the messages that prints with this (well, unless you add one of those words to the highlight list), so not sure we need this separate taglist?

Reply:
team messages (the "team" version) are not even marked in my weechat (I don’t know if I did something about it or if it came that way), so I only see them if I think of going to the team buffer.

These ones are "more important" than the usual debug messages, so I guess they should be marked as important? Otherwise there is no point in notifying...

Given that, I think there should be a difference between the "debug" messages tags and the "important" messages (maybe highlight isn’t the right term, I have no opinion about that), and as far as I understand tags are the way to do it in weechat (?)

Reply:
Hm, the core buffer appears in my hotlist when we print to it. I guess it depends on some weechat settings. But sure, I agree that the messages are different. Can you rename `team_highlight` to `team_message` and `team` to `team_info` though?

Reply:
:+1:

Reply:
✓